BackofficeBundle/Resources/public/coffee/backboneRouter.coffee

```diff
@@ -7,6 +7,9 @@ OrchestraBORouter = Backbone.Router.extend(
 
   initialize: ->
     @generateRoutePatterns()
+    jQuery ->
+      ribbonFormButtonViewClass = appConfigurationView.getConfiguration('ribbon-form-button', 'createRibbonFormButton')
+      OpenOrchestra.RibbonButton.RibbonFormButtonView.ribbonButton = new ribbonFormButtonViewClass()
```

Comment:
upperCase for the class, lowerCase for instance. So OpenOrchestra.RibbonButton.RibbonFormButtonView for the class and OpenOrchestra.RibbonButton.ribbonFormButtonView for the instance.